src/mixins/themeable.ts

```diff
@@ -39,108 +51,170 @@ export interface ThemeableOptions {
 	properties: ThemeableProperties;
 }
 
+export interface ClassesChain {
```

Comment:
Second thoughts I actually `Chain` in the name but also think it should contain `Function`, especially as there is no jsdoc, how about `ClassesFunctionChain`?